pkg/kube/daemonset.go

```diff
@@ -54,6 +55,8 @@ func GenerateDaemonSet(instance *sfv1alpha1.SplunkForwarder) *appsv1.DaemonSet {
 		volumes = GetVolumes(true, true, instance.Name)
 	}
 
+	priority := int32(2000001000)
```

Comment:
If this needs to be an `int32` we should set it in the variable declaration instead of casting:
```suggestion
	var priority int32 = 2000001000
```

Reply:
Updated per your suggestion.